File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int left=1;
        for(int i=0; i<nums.length;i++) {
            if(i>0 && nums[i]!=nums[i-1]){
                nums[left++]=nums[i];
            }
        }
        return left;
        
    }
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums)==0: return 0
        left=1
        for i in range(len(nums)):
            if i>0 and nums[i]!=nums[i-1]: 
                nums[left]=nums[i]
                left+=1
        return left